Model/EventsForThirdPartyModules.php

```diff
@@ -0,0 +1,122 @@
+<?php
+/**
+ * Bolt magento2 plugin
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * http://opensource.org/licenses/osl-3.0.php
+ *
+ * @category   Bolt
+ * @package    Bolt_Boltpay
+ * @copyright  Copyright (c) 2017-2020 Bolt Financial, Inc (https://www.bolt.com)
+ * @license    http://opensource.org/licenses/osl-3.0.php  Open Software License (OSL 3.0)
+ */
+
+namespace Bolt\Boltpay\Model;
+
+use Bolt\Boltpay\Helper\Bugsnag;
+use Exception;
+
+class EventsForThirdPartyModules
+{
+    const eventListeners = [
+        "afterLoadSession" => [
+            "listeners" => [
+                [
+                    "module" => "Mageplaza_ShippingRestriction",
+                    "3pclasses" => ["Mageplaza\ShippingRestriction\Helper\Data"],
+                    "boltClass" => "Bolt\Boltpay\ThirdPartyModules\Mageplaza\ShippingRestriction",
```

Comment:
Can we instead use `ShippingRestriction::class` or similar?

Reply:
Ok I'll add 
use `Bolt\Boltpay\ThirdPartyModules\Mageplaza\ShippingRestriction as Mageplaza_ShippingRestriction` then